Scripts/Cast/PointsCast.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace VRTK.Core.Cast
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using System.Collections.Generic;
+    using VRTK.Core.Process;
+    using VRTK.Core.Utility;
+
+    /// <summary>
+    /// Contains information about the current <see cref="PointsCast"/> state.
+    /// </summary>
+    public class PointsCastData
+    {
+        /// <summary>
+        /// The result of the most recent cast. <see langword="null"/> when the cast didn't hit anything.
+        /// </summary>
+        public RaycastHit? TargetHit;
```

Comment:
Nullable because I don't want the intrinsic knowledge of how Unity uses `RaycastHit` to be part of this - they could change that, too (but probably won't). This way a user just has to test for existence, rather than know about how `RaycastHit` handles its properties to signal "nothing was hit".